pkgs/applications/virtualization/virt-manager/default.nix

```diff
@@ -9,10 +9,6 @@
 
 with stdenv.lib;
 
-# TODO: remove after there's support for setupPyDistFlags
-let
-  setuppy = ../../../development/interpreters/python/run_setup.py;
-in
 python3Packages.buildPythonApplication rec {
   name = "virt-manager-${version}";
```

Comment:
if you need to edit again, you could rename name to pname and discard the version suffix.

Reply:
Hm, is there some way to reference `name` to avoid writing `.../${pname}-${version}.tar.gz`?

Reply:
not that I know o. Forget it then sry :p